rest/route/agent.go

```diff
@@ -1645,6 +1653,11 @@ func (h *createGitHubDynamicAccessToken) Run(ctx context.Context) gimlet.Respond
 	permissions := &intersection.Permissions
 	if intersection.AllPermissions {
 		permissions = nil
+	} else if intersection.HasNoPermissions() {
+		return gimlet.MakeJSONErrorResponder(gimlet.ErrorResponse{
+			StatusCode: http.StatusUnauthorized,
+			Message:    "the intersection of the requester's permissions and provided permissions does not have permission to create a token",
```

Comment:
sorry, quick nit: 
Can we change the wording here to describe the intersection in terms of the command and the project settings so that it's more familiar to the user?